policyengine_canada/parameters/gov/cra/benefits/old_age_security_pension/amount/repayment_rate.yaml

```diff
@@ -0,0 +1,13 @@
+description: OAS Repayment Rate
+values:
+  2022-01-01: 0.15
+
+metadata:
+  type: single_amount
```

Comment:
Lines 6-8 only apply to [scale parameters](https://policyengine.github.io/policyengine-core/python_api/parameters.html#parameterscale), but I'd actually suggest using that here. It can be very similar to [this parameter](https://github.com/PolicyEngine/policyengine-canada/blob/master/policyengine_canada/parameters/gov/cra/benefits/cwb/phase_out/base/single.yaml). Could you make a file called `old_age_security_pension/repayment_tax.yaml` with that structure?

Reply:
Could you also adjust the dates to reflect the fiscal year, and add the other two years's thresholds specified in the CRA page?

Reply:
The OAS repayment is a bit different from the CWB reduction because the latter is calculated as a percentage of income (afni) once you pass a certain threshold, while the former is calculated as a percentage of _the difference_ between income and that threshold. So I'm having trouble thinking of a way to calculate the OAS reduction using only one scale parameter, without calculating the difference first using a separate `repayment_threshold.yaml` parameter, as I'd done initially. But I'm still learning. What do you think @MaxGhenis ?

From the CWB docs:

_"The benefit is phased out by 12% of adjusted family net income over $13,194* for single individuals without dependants and $17,522* for families."_

From the OAS docs: 

_"Your [repayment](https://www.canada.ca/en/services/benefits/publicpensions/cpp/old-age-security/repayment.html) calculation is based on the difference between your income and the threshold amount for the year. The first step is to figure out how much higher your income is than the threshold. You must repay 15% of that amount."_